src/ui/server/shared-validation/website/index.ts

```diff
@@ -0,0 +1,25 @@
+import isUrl from 'is-url';
+import generateValidationErrors from '../../helpers/validation';
+
+/**
+ * validates website string and adds errorMessage and fieldId to errors if not in the correct formap
+ * @param {string} website
+ * @param {string} fieldId
+ * @param {string} errorMessage
+ * @param {object} errors
+ * @returns {object} errors
+ */
+const validateWebsite = (website: string, fieldId: string, errorMessage: string, errors: object) => {
```

Comment:
Do you think we should be more descriptive with the file and function name? i think website is a little too generic, maybe could be `validateWebsiteAddress` and directory as `website-address`, what do you think?

Reply:
done